File: Source/JapaneseLanguageTools.Contracts/Models/Blobs/Responses/DownloadBlobResponse.Model.cs
using JapaneseLanguageTools.Contracts.Models.Blobs.Requests;

namespace JapaneseLanguageTools.Contracts.Models.Blobs.Responses;

public class DownloadBlobResponseModel
{
    public required BlobMetadataModel BlobMetadata { get; set; }

    public required DownloadBlobRequestModel Request { get; set; }
}

File: Source/JapaneseLanguageTools.ReactSpa/source/models/blobs/responses/DownloadBlobResponse.Model.ts
import BlobMetadataModel from "@/models/blobs/BlobMetadata.Model";
import DownloadBlobRequestModel from "@/models/blobs/requests/DownloadBlobRequest.Model";

export default interface DownloadBlobResponseModel {
  blobMetadata: BlobMetadataModel;

  request: DownloadBlobRequestModel;
}
